content/en/integrations/guide/azure-vms-appear-in-app-without-metrics.md

```diff
@@ -20,6 +20,5 @@ If after this time you see Azure VMs in your infrastructure list but no metrics
     {{< img src="integrations/guide/azure_vms_appearing_in_the_app_without_metrics/arm_deployed_vm.png" alt="The azure portal showing the diagnostics settings view of a virtual machine with status set to on" >}}
 
 3. Make sure the Virtual machine is running.
-    The integration does not collect performance metrics for stopped/deallocated machines. However, the `azure.vm.status metric` returns `1` if the machine is running, or stopped (which results in stopped VMs showing up in the infrastructure list). The associated status tag allows you to differentiate between running and not-running hosts. Make sure the host in question has `status:running`, and is running in the Azure portal.
-    {{< img src="integrations/guide/azure_vms_appearing_in_the_app_without_metrics/azure_vm_running.png" alt="a pair of timeseries graphs in the Datadog application, with one displaying the sum of azure.vm.status with status:running and the other displaying the sum of azure.vm.status with status:not_running" >}}
-
+    The integration does not collect performance metrics for stopped or deallocated machines. Use the `azure.vm.count metric` and the `status` tag values of `running`, `stopped`, and `stopped_deallocated` to determine the status of your hosts. Make sure the host in question has `status:running`, and is running in the Azure portal.
```

Comment:
Just wanted to double-check whether the word "metric" in `azure.vm.count metric` is meant to be formatted as code.

Reply:
Thanks for flagging that! Updated in [04990bf](https://github.com/DataDog/documentation/pull/29122/commits/04990bf921444311ce3c254df2d58a23f1813ae9)